Source/Config/Main.lua

```diff
@@ -0,0 +1,50 @@
+Auctionator.Config.Option = {
```

Comment:
Can we pluralize `Option`? It reads funny to me elsewhere when we're indexing into the table. 😁